### DISCUSSION TITLE
Ticketing progress bar dialog

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14940] [How to] [answered]
How to switch off confirmation dialog as per screenshot? It looks for me like unnecessary click :-)

![image](https://user-images.githubusercontent.com/84110093/125144872-867af780-e149-11eb-8ecb-316cfe7cd65f.png)

best

**Comment by rskrzypczak** [comment, marked as answer]
This cannot be turned off.
If you want to do that, you have to remove it from the source code, which I personally do not recommend.
By the way, I don't think it is a redundant step :).
